website/src/CONTRIBUTING.md

```diff
@@ -0,0 +1,309 @@
+# Contributing to OpenEBS Website
```

Comment:
We should move `CONTRIBUTING.md` to the root of this repo since here we are talking about the website as well as docs ( both the source code are different)

Reply:
somehow, images are broken

<img width="1151" alt="Screenshot 2021-06-23 at 9 00 33 AM" src="https://user-images.githubusercontent.com/6524419/123031199-91235600-d401-11eb-942a-1bc26b7f3218.png">

Reply:
FYI, Images will not load in this preview as PR is not merged, so it didn't get the exact path to load images.